src/lib.rs

```diff
@@ -769,6 +833,63 @@ macro_rules! __impl_bitflags {
         $(#[$filtered])*
         fn $($item)*
     };
+
```

Comment:
nit: there are a few extra newlines here.

Reply:
This is fixed now